docs/languages/julia.md

```diff
@@ -0,0 +1,71 @@
+# Julia in Visual Studio Code
+
+The Visual Studio Code extension for Julia includes built-in dynamic autocompletion, inline results, plot pane, integrated REPL, variable view, code navigation, and many other advanced language features.
```

Comment:
Make this a link to the extension.
Probably better to say "The [Julia](link to Marketplace) extension for Visual Studio Code provides [Julia](https://julialang.org) programming language support, which includes ...."